### ISSUE TITLE
Postfixadmin manage domains / emails

### ISSUE THREAD

**Issue by klevinkona** [postfixadmin/postfixadmin#150]
Hello there,
If i can ask since i know postfixadmin is webinterface only by default, is there any way how to change it to manage the emails / domains ?

**Comment by DavidGoodwin** [comment]
Can you rephrase your question please?

Postfixadmin allows you to add/remove domains, mailboxes within those domains and any forwards/redirects/aliases for those domains.

**Comment by klevinkona** [comment]
Hello Davis, i configured postfix and postfixadmin but when i create a user in postfix adimn i do not se it in the server mail direktori, usually should have created a user and a mailbox for the new user?
Am i right? I have missing something here.
The configuration is done in debian 9 verison

**Comment by DavidGoodwin** [comment]
I think when you create a mailbox, there's a tickbox which triggers an email to be sent for a newly created user.  Otherwise the mailbox is only created when they first receive mail.

**Comment by klevinkona** [comment]
Yes, but i am asking if except the basic configuration there are other configuration to be done? because i see the account created in postfixadmin does not apply to the real account emails.
How should i connect postfixadmin with postfix to interact?
